tests/python/contrib/test_ethosu/test_tir_to_cs_translator.py

```diff
@@ -764,7 +764,4 @@ def check_buffer(address, region, length, buffer_var):
 
 
 if __name__ == "__main__":
-    test_buffer_info_extraction()
-    test_translate_ethosu_conv2d()
-    test_translate_ethosu_copy()
-    test_assign_addresses()
+    pytest.main([__file__])
```

Comment:
`sys.exit(pytest.main([__file__] + sys.argv[1:]))` please

Reply:
cc @denise-k can you add an issue to testing roadmap to provide a standard test boilerplate utility function?